Modules/Bridge/VtkGlue/include/itkImageToVTKImageFilter.h

```diff
@@ -105,8 +105,8 @@ class ITK_TEMPLATE_EXPORT ImageToVTKImageFilter : public ProcessObject
   ~ImageToVTKImageFilter() override;
 
 private:
-  ExporterFilterPointer m_Exporter;
-  vtkImageImport *      m_Importer;
+  ExporterFilterPointer m_Exporter{};
+  vtkImageImport *      m_Importer{};
```

Comment:
I assume both of these initialize to nullptr?

Reply:
> I assume both of these initialize to nullptr?

The short answer: yes 👍 

The long answer: `m_Importer` is a raw pointer, so it is initialized to nullptr, by `m_Importer{}`. m_Exporter is a ExporterFilterPointer, which is a `itk::SmartPointer<T>`. Which is default-constructed by `m_Exporter{}`.  The default-constructor of `itk::SmartPointer<T>` yields a smart pointer whose internal `m_Pointer` member is equal to nullptr.

Reply:
Niels already replied and made the point clear.

The SW Guide PR has already been opened.

Resolving.